leasing/models/rent.py

```diff
@@ -1476,6 +1476,118 @@ class LegacyIndex(models.Model):
     )
 
 
+class OldDwellingsInHousingCompaniesPriceIndex(models.Model):
+    """
+    In Finnish: Vanhojen osakeasuntojen hintaindeksi
+
+    From my understanding, a unique price index is identified by the API
+    database table URL and code of the index. A single table can hold multiple
+    indexes, and a single code can be used in multiple tables.
+
+    Members:
+        code: Code for the index's table column. Example: "ketj_P_QA_T". Same code for different intervals, e.g. yearly or quarterly.
+        name: Name of the index. Example: "Index (2020=100)".
+        comment: Comment for the index's table column.
+        source: Source of the data.
+        source_table_updated: UTC timestamp when the source table was last updated.
+        source_table_label: Label for the source table.
+        url: API endpoint URL.
+    """
+
+    # Maximum lengths are arbitrary, but set to avoid extra large input.
+    CHARFIELD_MAX_LENGTH = 255
```

Comment:
It might be that this is the default value used in Django, but does not hurt to define it so it is clear what it is